plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -81,6 +81,14 @@ class Offboard : public PluginBase {
                           above). */
     };
 
+// Start of code - Setting Atitude with angular rates the following sequence (Roll, Pitch, Yaw, Thrust).
```

Comment:
Let's remove the "Start of code" and "end of code" comments :-)

Reply:
Removed the Comments. 
Sorry about causing inconvenience.

Reply:
No worries, thank you ;-)